09-secrets.md

```diff
@@ -1,28 +1,33 @@
 ## Secrets top-level element
 
-Secrets are a flavour of Configs focusing on sensitive data, with specific constraint for this usage. As the platform implementation may significantly differ from Configs, dedicated Secrets section allows to configure the related resources.
+Docker Compose provides a way for you to use secrets without having to use environment variables to store information. If you’re injecting passwords and API keys as environment variables, you risk unintentional information exposure. Environment variables are often available to all processes, and it can be difficult to track access. They can also be printed in logs when debugging errors without your knowledge. Using secrets mitigates these risks.
```

Comment:
this is compose specification, not Docker Compose documentation. Should rephrase as `a Compose implementation` or similar

Reply:
I understand, I've changed it to just 'Compose' - very broad and general - as people reading this on docs.docker.com will be thrown with 'A Compose implementation' for example